wls-wahlvorbereitung-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/service/urnenwahlvorbereitung/UrnenwahlvorbereitungValidator.java

```diff
@@ -16,4 +16,11 @@ public void validWahlbezirkIDOrThrow(final String wahlbezirkID) {
             throw exceptionFactory.createFachlicheWlsException(ExceptionConstants.SUCHKRITERIEN_UNVOLLSTAENDIG);
         }
     }
+
```

Comment:
Add a comment describing what makes a UrnenwahlvorbereitsungModel invalid.

Reply:
i am not a fan of explicit comments that might get out of sync very easy. And i think the caller is not interested in the reason. He just want that is valid or some Exception will through.

Why do you think a comment is required?

Reply:
done with 546ac5b8fc972c7e4d70807ad4e748979d040a39